.flexci/linux/Dockerfile

```diff
@@ -28,11 +28,14 @@ ARG python_version
 RUN pyenv install ${python_version} && \
     pyenv global ${python_version}
 
+COPY ./setup.py ./pytorch_pfn_extras/setup.py
```

Comment:
`setup.py` is not always guaranteed to run without full source tree. How about placing `tests/requirements.txt` and declaring test dependencies there, instead of extra requirements?